web/themes/custom/jcc_components/includes/views.inc

```diff
@@ -390,6 +390,15 @@ function jcc_components_preprocess_views_view__documents__default(&$variables) {
 
     // Return published oral arguments if available.
     $latest_oa = array_slice($oral_args, 0, 1);
+    if($latest_oa) {
+      $variables['show_documents'] = FALSE;
```

Comment:
394 is not necessary because if the variable does not have a value TWIG will default to NULL or FALSE.

$variables[''] is a way to pass new variables for TWIG use so it's not necessary to define an initial value in PHP to use either.

Reply:
Yes index[0] was needed here

Reply:
Removed, thanks!